test/ZNSAddressResolver.test.ts

```diff
@@ -1,195 +1,232 @@
 
     // The domain does not exist
     const someDomainHash = hashDomainLabel("random-record");
-    const notExistResolver = await znsRegistry.getDomainResolver(someDomainHash);
+    const notExistResolver = await zns.registry.getDomainResolver(someDomainHash);
     expect(notExistResolver).to.eq(hre.ethers.constants.AddressZero);
   });
 
-  it("Should have registry address correctly set when initializing", async () => {
-    expect(await znsAddressResolver.registry()).to.equal(znsRegistry.address);
+  it("Should have registry address correctly set", async () => {
+    expect(await zns.addressResolver.registry()).to.equal(zns.registry.address);
   });
 
   it("Should setRegistry() correctly with ADMIN_ROLE", async () => {
     await expect(
-      znsAddressResolver.connect(deployer).setRegistry(operator.address)
+      zns.addressResolver.connect(deployer).setRegistry(operator.address)
     )
-      .to.emit(znsAddressResolver, "RegistrySet")
+      .to.emit(zns.addressResolver, "RegistrySet")
       .withArgs(operator.address);
 
-    expect(await znsAddressResolver.registry()).to.equal(operator.address);
+    expect(await zns.addressResolver.registry()).to.equal(operator.address);
   });
 
   it("Should revert when setRegistry() without ADMIN_ROLE", async () => {
     await expect(
-      znsAddressResolver.connect(operator).setRegistry(operator.address)
+      zns.addressResolver.connect(operator).setRegistry(operator.address)
     ).to.be.revertedWith(
       getAccessRevertMsg(operator.address, ADMIN_ROLE)
     );
   });
 
   it("Should setAccessController() correctly with ADMIN_ROLE", async () => {
-    await expect(
-      znsAddressResolver.connect(deployer).setAccessController(operator.address)
-    )
-      .to.emit(znsAddressResolver, "AccessControllerSet")
+    expect(await zns.addressResolver.connect(deployer).setAccessController(operator.address))
+      .to.emit(zns.addressResolver, "AccessControllerSet")
       .withArgs(operator.address);
 
-    expect(await znsAddressResolver.getAccessController()).to.equal(operator.address);
+    expect(await zns.addressResolver.getAccessController()).to.equal(operator.address);
   });
 
   it("Should revert when setAccessController() without ADMIN_ROLE", async () => {
     await expect(
-      znsAddressResolver.connect(operator).setAccessController(operator.address)
+      zns.addressResolver.connect(operator).setAccessController(operator.address)
     ).to.be.revertedWith(
       getAccessRevertMsg(operator.address, ADMIN_ROLE)
     );
   });
 
   it("Should not allow non-owner address to setAddress", async () => {
     await expect(
-      znsAddressResolver.connect(addr1).setAddress(wilderDomainNameHash, addr1.address)
-    ).to.be.revertedWith(
-      "ZNSAddressResolver: Not authorized for this domain"
-    );
+      zns.addressResolver.connect(user).setAddress(wilderDomainHash, user.address)
+    ).to.be.revertedWith("ZNSAddressResolver: Not authorized for this domain");

```

Comment:
why do you keep reformatting my code for small things like this? how many times will we format back and forth? :)
maybe we should create a guide or something and stick to it, if you are still not satisfied with how I format. Thought we've talked about this already :)
this is all just extra time for small things like this...